modules/installation-guide/partials/proc_importing-untrusted-tls-certificates.adoc

```diff
@@ -22,121 +25,90 @@ endif::[]
+- {prod-short} automatically labels the config map with the `config.openshift.io/inject-trusted-cabundle="true"` label.
+- Based on this annotation, OpenShift automatically injects the cluster-wide trusted CA certificates inside the `ca-bundle.crt` key of config map
 ====
 
-== At {prod-short} installation time
+== Adding new CA certificates into {prod-short}
+
+This guide can be used before the installations of {prod-short} or when {prod-short} is already installed and running.
+
+include::example$snip_importing-untrusted-tls-certificates_1.adoc[]
 
 .Prerequisites
 
 * The `{orch-cli}` tool is available.
-* You are ready to create `CheCluster` custom resource.
+* Namespace for {prod-short} exists.
 
 .Procedure
 
-include::example$snip_importing-untrusted-tls-certificates_1.adoc[]
-
-. Create a new ConfigMap with the required TLS certificates:
+. Create a new config map with the required TLS certificates:
 +
 [subs="+attributes,+quotes"]
 ----
-$ {orch-cli} create configmap ca-certs --from-file=__<certificate-file-path>__ -n=__<{project-context}-namespace-name>__
+$ {orch-cli} create configmap custom-certs --from-file=__<bundle-file-path>__ -n=__<{project-context}-namespace-name>__
 ----
 +
-To apply more than one certificate, add another `--from-file=_<certificate-file-path>_` option to the above command.
+To apply more than one bundle, add another `--from-file=_<bundle-file-path>_` option to the above command.
+Or you may create another config map.
 
-. During the installation process, when creating the `CheCluster` custom resource, take care of configuring the right name for the created ConfigMap.
-+
-====
-For a {prod-short} link:https://docs.openshift.com/container-platform/latest/operators/olm-what-operators-are.html[Operator] deployment,
-ensure you add the `spec.server.ServerTrustStoreConfigMapName` field with the name of the ConfigMap, to the `CheCluster` Custom Resource you will create during the installation:
-
-[source,yaml,subs="+quotes",options="nowrap",role=white-space-pre]
-----
-spec:
-  server:
-    ...
-    spec.server.ServerTrustStoreConfigMapName: ca-certs
-----
-====
-ifeval::["{project-context}" == "che"]
+. Label created config maps with both `app.kubernetes.io/part-of=che.eclipse.org` and `app.kubernetes.io/component=ca-bundle` labels:
 +
-====
-For a {prod-short} link:https://helm.sh/[Helm Chart] deployment, ensure you override the `global.tls.serverTrustStoreConfigMapName` Helm Chart property with the name of the ConfigMap when installing the {prod-short} Helm Chart. For this you should add the following arguments to the Helm command line:
-[subs="+quotes",options="nowrap",role=white-space-pre]
+[subs="+attributes,+quotes"]
 ----
---set global.tls.serverTrustStoreConfigMapName=ca-certs
+$ {orch-cli} label configmap custom-certs app.kubernetes.io/part-of=che.eclipse.org -n che && kubectl label configmap custom-certs app.kubernetes.io/component=ca-bundle -n che
```

Comment:
{orch-cli} label configmap custom-certs -n __<{project-context}-namespace-name>__  app.kubernetes.io/part-of=che.eclipse.org app.kubernetes.io/component=ca-bundle